website/docs/r/one_dashboard.html.markdown

```diff
@@ -328,6 +329,9 @@ Each widget type supports an additional set of arguments:
         * `from` - The value 'from' which the threshold would need to be applied.
         * `to` - The value until which the threshold would need to be applied.
         * `severity` - The severity of the threshold, which would affect the visual appearance of the threshold (such as its color) accordingly. The value of this attribute would need to be one of the following - `warning`, `severe`, `critical`, `success`, `unavailable` which correspond to the severity labels _Warning_, _Approaching critical_, _Critical_, _Good_, _Neutral_ in the dropdown that helps specify the severity of thresholds in table widgets in the UI, respectively.
+    * `initial_sorting` - (Optional) An attribute that describes the sorting mechanism for the table. This attribute requires specifying the following attributes in a nested block -
+        * `name` - (Required) The name of column to be sorted. Examples of few valid values are `timestamp`, `appId`, `appName`, etc.
+        * `direction` - (Required) Defines the sort order. Accepted values are `asc` for ascending or `desc` for descending.
```

Comment:
@gmanandhar-nr similar to the other comment on this file: please add this to the table widget (one of them, if there is more than one table) in the sample we have at the beginning of this file